src/agni/server/AgniServer.java

```diff
@@ -1,10 +1,12 @@
 package agni.server;
 
-public class AgniServer {
-
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
+import java.nio.channels.SocketChannel;
+import java.util.ArrayList;
 
-	}
+public class AgniServer {
+        ArrayList<IPChannelPair<String,SocketChannel>> channels; 
```

Comment:
A map would be better suited in this case, because the most common use case we'll have is looking for one value based on one key (random access). An arraylist would be better if we'll be iterating through the list often.